ts/output/chtml/FontData.ts

```diff
@@ -188,6 +188,24 @@ export class ChtmlFontData extends FontData<ChtmlCharOptions, ChtmlVariantData,
     data.fonts && this.addDynamicFontCss(this.defaultStyles, data.fonts, data.fontURL);
   }
 
+  /**
+   * @override
+   */
+  public addExtension(
+    data: ChtmlFontExtensionData<ChtmlCharOptions, ChtmlDelimiterData>,
+    prefix: string = ''
+  ): string[] {
+    super.addExtension(data, prefix);
+    if (data.fonts && this.options.adaptiveCSS) {
+      const css = {};
+      const styles = new CssStyles();
+      (this.constructor as typeof ChtmlFontData).addDynamicFontCss(css, data.fonts, data.fontURL);
+      styles.addStyles(css);
+      return styles.getStyleRules();
+    }
+    return [];
```

Comment:
Do that case first with the negated `if` condition and safe indentation for the main logic.